packages/manager/apps/dedicated/client/app/ip/components/list/byoip/constants.js

```diff
@@ -13,32 +13,32 @@ export const BYOIP_USAGE_GUIDE_URL = {
   IT:
```

Comment:
**question:** Why not for `IT`?

Reply:
Hello @anooparveti all the links should include KB**** since it is good we are not updating it.